native/react-native/common-components/button.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { StyleSheet, TouchableOpacity, Text, Image } from 'react-native';
+import PropTypes from 'prop-types';
+
+const styles = StyleSheet.create({
```

Comment:
What are you guys' thoughts on using StyledComponents (https://github.com/styled-components/styled-components)? I used them and I like the flexibility of programmatically updating the values and the fact that I can re-use the styles since it becomes a Component.

Reply:
For discussion, here's the docs for specifically React Native https://www.styled-components.com/docs/basics#react-native
What do you think @j-ornelas @kmausolf @Fleury14 ?

Reply:
I feel like what this does could be useful, but it's heavily invested in the idea that we style native components with CSS.  Not only is this notion incorrect, it also tempts bad habits from CSS that may or may not translate to native code.  I think this is a cool library, but it doesn't belong in a standardized method of doing things in RN.

Reply:
Ah, I see what you mean. They do claim to support React Native (https://web.archive.org/web/20190127113205/https://www.styled-components.com/docs/basics) - I keep getting DSN issues with their site - which allows you to wrap the Native components and style them using the Native properties.